docs/source/io_formats/mgxs_library.rst

```diff
@@ -8,297 +8,155 @@ OpenMC can be run in continuous-energy mode or multi-group mode, provided the
-    requested and the material is fissionable.
-
-  :kappa_fission:
-    This element requires the group-wise kappa-fission cross section ordered by
-    increasing group index (i.e., fast to thermal).  If ``representation`` is
-    "isotropic", then the length of this list should equal the number of
-    groups described in the ``groups`` element.  If ``representation`` is
-    "angle", then the length of this list should equal the number of groups
-    times the number of azimuthal angles times the number of polar angles,
-    with the inner-dimension being groups, intermediate-dimension being
-    azimuthal angles and outer-dimension being the polar angles.
-
-    *Default*: None, this is required only if kappa_fission tallies are
-    requested and the material is fissionable.
-
-  :chi:
-    This element requires the group-wise fission spectra ordered by
-    increasing group index (i.e., fast to thermal).  This element should be
-    used if making the common approximation that the fission spectra does
-    not depend on incoming energy.  If the user does not wish to make this
-    approximation, then this should not be provided and this information
-    included in the ``nu_fission`` element instead.  If ``representation`` is
-    "isotropic", then the length of this list should equal the number of
-    groups described in the ``groups`` element.  If ``representation`` is
-    "angle", then the length of this list should equal the number of groups
-    times the number of azimuthal angles times the number of polar angles,
-    with the inner-dimension being groups, intermediate-dimension being
-    azimuthal angles and outer-dimension being the polar angles.
-
-    *Default*: None, either this element is provided or ``nu_fission`` is
-    provided in fission matrix form, or the material is not fissionable.
-
-  :nu_fission:
-    This element provides either the group-wise fission production cross
-    section vector (i.e., if ``chi`` is provided), or is the group-wise fission
-    production matrix.  If providing the vector, it should be ordered the same
-    as the ``fission`` data.  If providing the matrix, it should be ordered
-    the same as the ``multiplicity`` matrix.
-
-    *Default*: None, either this element must be provided if the material
-    is fissionable.
-
+**/**
+
+:Attributes: - **groups** (*int*) -- Number of energy groups
+             - **group structure** (*double[]*) -- Monotonically increasing
+               list of group boundaries, in units of MeV.  The length of this
+               array should be the number of groups plus 1.
+
+**/<library name>/**
+
+The data within <library name> contains the temperature-dependent multi-group
+data for the nuclide or material that it represents.
+
+:Attributes: - **awr** (*double*) -- The atomic weight ratio (optional, i.e. it
+               is not meaningful for material-wise data)
+             - **fissionable** (*int*) -- Whether the dataset is fissionable
```

Comment:
Can't the code easily determine whether a material is fissionable by inspection of the fission MGXS?